docs/pages/versions/unversioned/workflow/configuration.md

```diff
@@ -52,7 +52,9 @@ Valid values: `public`, `unlisted`
 
 ### `"sdkVersion"`
 
-**Required**. The Expo sdkVersion to run the project on. This should line up with the version specified in your package.json.
+__Deprecated__. The Expo sdkVersion to run the project on. This should line up with the version specified in your package.json.
```

Comment:
This is now optional, we still require it for `UNVERSIONED`. Also I think @dsokal ran into some problems migrating turtle. Regardless we are still working on some of the smaller details regarding app.config.

Reply:
Sorry, I might have been to fast with this. I'll close this for now. I'll make a short checklist for me to double-check the documentation once that's done

Reply:
Actually, the only problem with Turtle was that turtle-cli couldn't find the `sdkVerison` field in `app.json`. It's later used for determining which shell app to pick. Other than that, it's not impacting Turtle. I fixed the issue by using `@expo/config` for finding the right version.